modules/projects/server/models/project.model.js

```diff
@@ -142,7 +142,7 @@ module.exports = require ('../../../core/server/controllers/core.schema.controll
   projectNotes    : { type: String, default: '' }, // Formal notes about the project
 
   // MMTI hook-up data
-  isMajorMine      : { type: Boolean, default: false },
+  isMajorMine      : { type: Boolean, default: false, index: true },
```

Comment:
Will you be adding the index manually on the various environments?

Reply:
Already done last week. ;)

Reply:
Awesome! 👍